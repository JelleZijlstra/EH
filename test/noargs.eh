#!/usr/bin/ehi
# What happens when we call a function with no arguments?
$echo printvar:
$echo is_int:
