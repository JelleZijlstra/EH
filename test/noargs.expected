Incorrect argument count for function printvar: expected 1, got 0: EH runtime error
(null)
Incorrect argument count for function is_int: expected 1, got 0: EH runtime error
(null)
